composer.json

```diff
@@ -53,7 +53,6 @@
         "symfony/yaml": "^4.4|^5.0",
         "symfony/security-bundle": "^4.4|^5.0",
         "symfony/web-profiler-bundle": "^4.4|^5.0",
-        "symfony/twig-bundle": "^4.4|^5.0",
```

Comment:
@Th3Mouk TwigBundle is still enabled in some tests, f.e. https://github.com/Th3Mouk/FOSRestBundle/blob/fix/without-twig/Tests/Functional/app/Version/bundles.php#L14

Even after this change `symfony/twig-bundle` is still installed since it is a dependency of `symfony/web-profiler-bundle`. 
I guess that's the reason why the tests are still green.

Furthermore from the test name I assume that https://github.com/Th3Mouk/FOSRestBundle/blob/fix/without-twig/Tests/Functional/RequestBodyParamConverterTest.php#L35 is related to TwigBundle.

Therefore I guess we need to keep the dev dependency but should just not enable the bundle for all test cases **except** https://github.com/Th3Mouk/FOSRestBundle/blob/fix/without-twig/Tests/Functional/app/RequestBodyParamConverterTwigBundle/bundles.php#L14

WDYT?

Reply:
@W0rma I upgraded the PR, i tried to understand the remaining test case and see if i could add a missing one but all is ok, all seems covered